lib/friendly_id/sequentially_slugged.rb

```diff
@@ -7,11 +7,17 @@ def self.setup(model_class)
     def resolve_friendly_id_conflict(candidate_slugs)
       candidate = candidate_slugs.first
       return if candidate.nil?
-      SequentialSlugCalculator.new(scope_for_slug_generator,
+      scope = scope_for_slug_generator
```

Comment:
this could probably go back to being used inline?

```
SequentialSlugCalculator.new(scope_for_slug_generator,

Reply:
@parndt done